framework/scenario/src/scenario/run_vm/tx_output_check.rs

```diff
@@ -51,20 +70,33 @@ pub fn check_tx_output(tx_id: &str, tx_expect: &TxExpect, tx_result: &TxResult)
                     }
                 } else if !expected_logs.more_allowed_at_end {
                     panic!(
-                        "Unexpected log. Tx id: '{}'. Index: {}.\nAddress: {}, Endpoint: {}, Topics: {:?}, Data: {}",
-                        tx_id,
-                        i,
-                        address_hex(&actual_log.address),
-                        &actual_log.endpoint,
-                        verbose_hex_list(actual_log.topics.as_slice()),
-                        verbose_hex_list(&actual_log.data),
+                        "{}",
+                        unexpected_log(
+                            tx_id,
+                            i,
+                            address_hex(&actual_log.address),
+                            &actual_log.endpoint,
+                            verbose_hex_list(actual_log.topics.as_slice()),
+                            verbose_hex_list(&actual_log.data)
+                        ),
                     )
                 }
             }
         },
     }
 }
 
+fn format_result_message(message: &CheckValue<BytesValue>) -> String {
+    let mut formatted_message = message.to_string();
```

Comment:
not like that

use trim() to get rid of the quotes
.strip_prefix("str") to get rid of the prefix

Reply:
funny enough, `trim()` works only for whitespaces, but not quotes. 

I sanitized a bit the code tho